common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt

```diff
@@ -273,6 +273,42 @@ d_burning_blade = {
 	}
```

Comment:
Your CoAs should be a bit higher, there's a mask in the guidelines. The red dot represents the center of the image.
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/2D-Artist-Guidelines
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/123698376-2776d200-d86f-11eb-9cfa-53d5a0ae624c.png)

</details>

Reply:
Resolved by #498.